lib/Doctrine/DBAL/Logging/BacktraceLogger.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Doctrine\DBAL\Logging;
+
+class BacktraceLogger extends DebugStack
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function startQuery($sql, ?array $params = null, ?array $types = null)
+    {
+        parent::startQuery($sql, $params, $types);
+
+        $this->queries[$this->currentQuery]['backtrace'] = array_reverse(debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS));
```

Comment:
should we actually reverse the backtrace ? This could be confusing for users, as it would display it in the reverse order compared to what they are used to.

Reply:
I agree. Backtraces always show the most recent call first and we should keep it that way.

Reply:
agree, removed 👍